### PR TITLE
feat(api): add swagger

### PR DESCRIPTION
**Pull request by Tezno**

### CODE REVIEW

**Comment by vlamy** [comment]
Cool ça 👍

**Comment by Tezno** [comment]
Result:

File: src/cmd/api/controllers/videos_list_handler.go [hunk 1]
```diff
@@ -10,18 +10,26 @@ import (
 )
 
 type VideoInfo struct {
-	Id    string `json:"id"`
-	Title string `json:"title"`
+	Id    string `json:"id" example:"1"`
+	Title string `json:"title" example:"mon titre"`
```
Comment (RaboliotLeGris) [inline src/cmd/api/controllers/videos_list_handler.go:14]:
```suggestion
	Title string `json:"title" example:"my title"`
```

File: src/cmd/api/docs/swagger.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
+{
```
Comment (RaboliotLeGris) [inline src/cmd/api/docs/swagger.json:1]: It will be awesome if we can find a way to check in the CI that the file generated with swagger is up to date.

**Review by RaboliotLeGris — APPROVED** [review]

**Pull request merged**